src/aiortc/rtcrtpsender.py

```diff
@@ -175,6 +176,14 @@ def replaceTrack(self, track: Optional[MediaStreamTrack]) -> None:
         else:
             self._track_id = str(uuid.uuid4())
 
+    @property
+    def _enabled(self) -> bool:
+        return self.__enabled
+
+    @_enabled.setter
+    def _enabled(self, enabled: bool) -> None:
+        self.__enabled = enabled
+
```

Comment:
Sorry I should have been clearer here: I don't think we need getter / setters at all, just initial the property as `self._enabled = True` in the constructor (`__init__`).